templates/servicenow.yaml.epp

```diff
@@ -1,6 +1,7 @@
 <%- | String $instance,
       String $user,
       String $password,
+      String $oauth_token,
```

Comment:
You'll want to add this somewhere at https://github.com/puppetlabs/puppetlabs-servicenow_cmdb_integration/blob/master/templates/servicenow.yaml.epp#L12 so that the oauth_token gets added to the config file

Reply:
Thanks for the review, updated!